apollo-ast/src/main/resources/apollo.graphqls

```diff
@@ -13,16 +12,9 @@ directive @optional(if: Boolean = true) on FIELD | VARIABLE_DEFINITION
 # and null in the other
 directive @nonnull(fields: String! = "") on OBJECT | FIELD
 
-# Whether to make all variables optional at the schema level
-# This is false by default. This can overriden with @optional
-directive @variables(optional: Boolean = false) on SCHEMA
```

Comment:
Long term, I'd like to move more of the Gradle configuration to `extra.graphqls` because:

1. it'll simplify the Gradle plugin
2. it's more granular (allows to target individual types, fields, etc...)

No need to keep stuff that's not working yet so 👍

Reply:
Makes sense!

Reply:
Another argument is:

3. makes it easier to use other build systems than Gradle (maven, buck, bazel, etc..)